docs: Create tasks for a relation and complete the task (#906)
anitab-org/mentorship-android: docs/manual_tests/Workflow.md

> # Workflow of App
> 
> ## Create tasks for a relation and complete the task

Reviewer: ```suggestion
# Create tasks in a mentorship relation and complete the task workflow
```

Steps

1. Registration/SignUp in app
1. LogIn in app
1. Navigating to members section
1. Sending request to mentor/mentee
1. Viewing request
1. Accepting/Rejecting request
1. Creating task

> 1. Task completion

Reviewer: I know this gets rendered correctly, but it would be nice to have it well numbered in the raw markdown as well.
1. ...
2. ...
(...)
